### DISCUSSION TITLE
DeDRM_plugin_ReadMe.txt (apprenticeharper)

### DISCUSSION THREAD

**Discussion by gracieclaire** [apprenticeharper/DeDRM_tools#1934] [Q&A]
hi!!! first off, thank you so much for all the helpful tips already provided. However, I am just so confused about where/how to find the "Kindle's serial number in the customization dialog."

I have kindle unlimited but do not have a Kindle device. I just use my mac computer or iPad. I am having so much trouble with converting the kindle books I bought from Amazon and downloaded to my computer and iPad to pdf. 

Also, once I find the kindle serial number, where do I find the customization dialog? I have downloaded all the files from the DeDRM_plugin_Readme.txt. and placed into calibre. 

I know this is a lot and I am most likely making no sense, but hopefully, you can help me :)

**Comment by ElleKayEm** [comment]
If you don't have an E-ink Kindle, then you don't need to worry about adding a serial number to DeDRM's customization.

You will need to download your books with Kindle for Mac  installed on the same machine and user as calibre with DeDRM.  See this post for getting books you can decrypt:  https://www.mobileread.com/forums/showthread.php?t=283371

FYI, these tools are intended for purchased books, not the loaned Kindle Unlimited books.
